packages/falcosidekick/config.yaml

```diff
@@ -0,0 +1,29 @@
+listenaddress: "" # ip address to bind falcosidekick to (default: "" meaning all addresses)
+listenport: 2801 # port to listen for daemon (default: 2801)
+debug: false # if true all outputs will print in stdout the payload they send (default: false)
+customfields: # custom fields are added to falco events, if the value starts with % the relative env var is used
+  # Akey: "AValue"
+  # Bkey: "BValue"
+  # Ckey: "CValue"
+templatedfields: # templated fields are added to falco events and metrics, it uses Go template + output_fields values
+  # Dkey: '{{ or (index . "k8s.ns.labels.foo") "bar" }}'
+# bracketreplacer: "_" # if not empty, replace the brackets in keys of Output Fields
+# TLS
+#  mutualtlsfilespath: "/etc/certs" # folder which will used to store client.crt, client.key and ca.crt files for mutual tls for outputs, will be deprecated in the future (default: "/etc/certs")
+#  mutualtlsclient: # takes priority over mutualtlsfilespath if not emtpy
+#    certfile: "/etc/certs/client/client.crt" # client certification file
+#    keyfile: "/etc/certs/client/client.key" # client key
+#    cacertfile: "/etc/certs/client/ca.crt" # for server certification
+#  tlsclient:
+#    cacertfile: "/etc/certs/client/ca.crt" # CA certificate file for server certification on TLS connections, appended to the system CA pool if not empty
+#  tlsserver:
+#    deploy: false # if true, TLS server will be deployed instead of HTTP
+#    certfile: "/etc/certs/server/server.crt" # server certification file
+#    keyfile: "/etc/certs/server/server.key" # server key
+#    mutualtls: false # if true, mTLS server will be deployed instead of TLS, deploy also has to be true
+#    cacertfile: "/etc/certs/server/ca.crt" # for client certification if mutualtls is true
+#    notlsport: 2810 # port to serve http server serving selected endpoints (default: 2810)
+#    notlspaths: # if not empty, and tlsserver.deploy is true, a separate http server will be deployed for the specified endpoints
+#      - "/ping"
+#      # - "/metrics"
+#      # - "/healthz"
```

Comment:
Can you embed the example config file? https://github.com/falcosecurity/falcosidekick/blob/master/config_example.yaml